### PR TITLE
#3: Create DB Schema for Job Portal

### PR DESCRIPTION
**Pull request by daneshpatel**

### CODE REVIEW

File: backend/db_schema/CJP_Schema.sql [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+CREATE TABLE "education" (
+  "id" SERIAL PRIMARY KEY,
+  "name" varchar
+);
+
+CREATE TABLE "user_education" (
+  "id" SERIAL PRIMARY KEY,
+  "user_id" int,
+  "employer_id" int,
+  "is_current_education" boolean
```
Comment (palnabarun) [inline backend/db_schema/CJP_Schema.sql]: Can we add two unique constraints on the following?

1. `(user_id, employer_id)`
2. `(user_id, is_current_education)`

File: backend/db_schema/CJP_Schema.sql [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+CREATE TABLE "skill" (
+  "id" SERIAL PRIMARY KEY,
+  "name" varchar
+);
+
+CREATE TABLE "user_skill" (
+  "id" SERIAL PRIMARY KEY,
+  "user_id" int,
+  "skill_id" int,
```
Comment (palnabarun) [inline backend/db_schema/CJP_Schema.sql:66]: Can we add a unique constraint on `(user_id, skill_id)`?

File: backend/db_schema/CJP_Schema.sql [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  "notice_period" varchar,
+  "resume" varchar,
+  "preferred_location" varchar,
+  "age" int
+);
+
+CREATE TABLE "user_employer" (
+  "id" SERIAL PRIMARY KEY,
+  "user_id" int,
+  "employer_id" int,
```
Comment (palnabarun) [inline backend/db_schema/CJP_Schema.sql:48]: Can we add a unique constraint on (user_id, employer_id)?

**Review by palnabarun — CHANGES_REQUESTED** [review]
Just some nits. Everything looks good otherwise.

File: backend/db_schema/CJP_Schema.sql (hunk 1 above)
Comment (daneshpatel) [reply backend/db_schema/CJP_Schema.sql]: SQL will not be going to execute on a server it just references for DBdiagram (Exported from dbdiagram)

(the table will be created by Django Models)

**Comment by inovizz** [comment]
Looks good to me.

@palnabarun - Shall we merge this, if you are good with this?

File: backend/db_schema/CJP_Schema.sql (hunk 1 above)
Comment (palnabarun) [reply backend/db_schema/CJP_Schema.sql]: Yeah. I guessed the same too. I added the comments for completeness. :)

**Comment by palnabarun** [comment]
@inovizz -- LGTM from my side. :+1: 

Thanks @daneshpatel for adding this.

**Review by palnabarun — APPROVED** [review]

**Pull request merged**